internal/cli/push.go

```diff
@@ -84,27 +86,29 @@ var CmdPush = &cobra.Command{
 		if len(args) == 1 {
 			path, err = os.Getwd()
 			if err != nil {
-				return errors.Wrap(err, "error pushing app")
+				return errors.Wrap(err, "working directory not accessible")
 			}
 		} else {
 			path = args[1]
 		}
 
 		if _, err := os.Stat(path); err != nil {
+			// Path issue is user error. Show usage
+			cmd.SilenceUsage = false
 			return errors.Wrap(err, "path not accessible")
 		}
 
 		i, err := instances(cmd)
```

Comment:
This is also a user error. You can un-silence the usage in the `instances()` function too.

Reply:
or do that inside the block below (like you did above too). There is another call to `instances()` here: https://github.com/epinio/epinio/pull/539/files#diff-3bc90f8ea54bc289631b44849d75d342573625452a6fde0144e7e2d98be80725R159

if you don't un-silence inside the `instances` method then you need to handle that there explicitly too.

Reply:
Done in `instances()`, commit 0a3700f9b1bf969e7270d4930d862e73a7a146da